doc/releases/release-notes-4.0.rst

```diff
@@ -377,6 +406,9 @@ Drivers and Sensors
 
 * Pin control
 
+  * STM32: :kconfig:option:`CONFIG_PIN_CONTROL` is now selected by drivers requiring it and
```

Comment:
```suggestion
  * STM32: :kconfig:option:`CONFIG_PINCTRL` is now selected by drivers requiring it and
```

Reply:
I knew there was something off with this :). Fixed